docs/02-app/01-building-your-application/05-optimizing/01-images.mdx

```diff
@@ -18,7 +18,9 @@ related:
 
 </details>
 
-The Next.js Image component extends the HTML `<img>` element with:
+According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/media#images) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
```

Comment:
```suggestion
According to [Web Almanac](https://almanac.httparchive.org), images account for a huge portion of the typical website’s [page weight](https://almanac.httparchive.org/en/2022/page-weight#content-type-and-file-formats) and can have an sizable impact on your website's [LCP performance](https://almanac.httparchive.org/en/2022/performance#lcp-image-optimization).
```

This link seems wrong. Seems like it should link to the page weight graph instead?